content/features/featuresDeepDive/mesh/gaussianSplatting.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Gaussian Splatting
+image: 
+description: Learn how use Gaussian Splatting.
+keywords: diving deeper, meshes, gaussian, splatting, nerf
+further-reading:
+video-overview:
+video-content:
+---
+
+# What is Gaussian Splatting?
+
+Gaussian Splatting is a volume rendering method. It's useful to capture real-life data. The difference with other technics like photogrammetry is the end result consists in a point cloud with each point rendered as a semi transparent ellipsoid projected onto a billboard. Gaussian Splatting is more suitable for rendering reflective and transparent surfaces.
+
+# Supported formats
+
+Supported formats are :
+- .PLY https://en.wikipedia.org/wiki/PLY_(file_format)
+- .splat that is Javascript types serialized version of .PLY datas
+
+# Loading a Gaussian Splatting
+
+Instanciate a new `GaussianSplattingMesh` object. It inherits from Mesh class. Set its name and the scene it belongs to.
+```javascript
+var gs = new BABYLON.GaussianSplattingMesh("Halo", scene);

```

Comment:
Should be: `new BABYLON.GaussianSplattingMesh("Halo", null, scene);`

You can also directly pass the `url` of the file instead of null, in which case you don't have to call `loadFileAsync` yourself.

Reply:
Fixed